colob_blob_tracker:
  lower_hue_lb: [90,50,50]
  lower_hue_ub: [120,255,255]
  upper_hue_lb: [90,50,50]
  upper_hue_ub: [120,255,255]
